### PR TITLE
Working on ttl file, moved around text

### PR DESCRIPTION
**Pull request by fosterlynn**
No mergy please!!  I'm doing this PR so people can look at my humble addition to the ttl file.  I added comments from the other doc; and added a bunch of properties (the ones that are datatype properties or reference non vf stuff).  Moved all of the text that wasn't a rdfs:comment into more general files, although it is not really integrated yet, just parked.

This ttl file won't load in webvowl; but it does seem to validate.  So I'm not sure what is wrong with it, but I'm sure something.  I also would like a review, in particular I'm not sure how to do the ranges of the datatype-y properties.  But also in general.

And I have some temporary comments when we have more than one non-hierarchical class as domain.  I fixed a few of those, but more to be figured out.

And I'm sure I'm still missing some properties, I just made sure all the ones from the existing doc got in there.

### CODE REVIEW

File: docs/introduction/resources.md [hunk 1]
```diff
@@ -22,6 +22,15 @@ is a model. Your library may have two copies that you can check out. Those are r
 
 *Names TBD.
```
Comment (pmackay) [inline docs/introduction/resources.md]: Is this a unclosed italic? Should it have a `*` at the end as well? Everything after is appearing as italics :)

File: docs/introduction/resources.md [hunk 2]
```diff
@@ -22,6 +22,15 @@ is a model. Your library may have two copies that you can check out. Those are r
...
 
+Resource categories are a Taxonomy. That means they can be defined very broadly and generally and maybe vaguely, or they can be defined very narrowly, but fit into broader categories. 
+
+So, for example, you may want an apple. Or you may want a green apple. Or you may want a Granny Smith apple.
+Or you may want a Granny Smith apple from your neighboring farm.
```
Comment (pmackay) [inline docs/introduction/resources.md]: Is this last one a valid example? I dont think the location is relevant for the resource category.

**Comment by fosterlynn** [comment]
@pmackay fixed, thanks!

**Comment by fosterlynn** [comment]
Made another commit.  Removed Intent from the ttl and uml, I think it will be a bit before we have that fleshed out.  And did a few other misc. things.

I'd like to work towards merging this, will wait a couple days for any further review / objections.  And won't merge until I can get a vowl diagram to work.

And may get the generated doc for classes and properties in there (from ttl file); if not, will do right after, so you can ignore all the class and property definition files, the ttl file is the important one.

File: docs/introduction/core.md [hunk 3]
```diff
@@ -1,17 +1,37 @@
...
+* vf:use - for example a tool used in process, after the process, the tool still exists
+* vf:consume - for example an ingredient composed into the output, after the process the ingredient is gone
+* vf:cite - for example a design file, neither used nor consumed, the file remains available at all times
+* vf:work - labor power towards a process
+* vf:accept - in processes like repair or maintentance, same resource will appear in output with *vf:improve* verb
+* vf:load -  transported resource enters the process
+
+###### Output
+
+* vf:create - new resource created in that process
```
Comment (elf-pavlik) [inline docs/introduction/core.md]: I think we used *vf:produce* here to match *vf:consume*

**Comment by fosterlynn** [comment]
@elf-pavlik thanks, fixed "produce"

File: release-doc-in-process/all_vf.TTL [hunk 4]
```diff
@@ -6,255 +6,854 @@
...
+        rdfs:label      "image" .
+
+vf:url a owl:DatatypeProperty ;
+        rdfs:label      "url" .
+
+vf:name a owl:DatatypeProperty ;
+        rdfs:label      "name" ;
+        rdfs:comment    "An informal or formal textual identifier for an object. Does not imply uniqueness." .
+
+vf:note a owl:ObjectProperty ;
```
Comment (djodjoni) [inline release-doc-in-process/all_vf.TTL]: vf:note is declared twice causing vowl json converter to fail. One needs to be removed.

File: release-doc-in-process/all_vf.TTL [hunk 5]
```diff
@@ -6,255 +6,854 @@
...
+
+vf:url a owl:DatatypeProperty ;
+        rdfs:label      "url" ;
+        rdfs:domain     vf:Agent .
+
+vf:name a owl:DatatypeProperty ;
+        rdfs:label      "name" ;
+        rdfs:comment    "An informal or formal textual identifier for an object. Does not imply uniqueness." .
+
+vf:note a owl:DatatypeProperty ;
```
Comment (djodjoni) [inline release-doc-in-process/all_vf.TTL]: this is probably the one we need to remove, unless we want the range to be a datavalue and vf:note is a subproperty of skos:note. actually do we need vf:note ?

File: release-doc-in-process/all_vf.TTL [hunk 6]
```diff
@@ -6,255 +6,854 @@
...
+        rdfs:label      "note" ;
+        rdfs:range      skos:note ;
+        rdfs:comment    "A textual comment or description." .
+
+vf:trackingIdentifier a owl:DatatypeProperty ;
```
Comment (djodjoni) [inline release-doc-in-process/all_vf.TTL]: DatatypeProperty could limit future uses, as ID could be represented by an individual resource. something like skos labels  had suffered from and skos-xl emerged to solve this.

File: release-doc-in-process/all_vf.TTL [hunk 7]
```diff
@@ -6,255 +6,854 @@
...
+vf:duration a owl:ObjectProperty ;
+        rdfs:label     "duration" ;
+        rdfs:domain     vf:EconomicEvent ;
+        rdfs:comment   "The observed duration of the economic event." .
+
+vf:image a owl:DatatypeProperty ;
+        rdfs:label      "image" ;
+        rdfs:domain     vf:Agent .
+
+vf:url a owl:DatatypeProperty ;
```
Comment (djodjoni) [inline release-doc-in-process/all_vf.TTL]: use foaf?

File: release-doc-in-process/all_vf.TTL [hunk 8]
```diff
@@ -6,255 +6,854 @@
...
+        rdfs:label      "url" ;
+        rdfs:domain     vf:Agent .
+
+vf:name a owl:DatatypeProperty ;
```
Comment (djodjoni) [inline release-doc-in-process/all_vf.TTL]: use foaf?

File: release-doc-in-process/all_vf.TTL [hunk 9]
```diff
@@ -6,255 +6,854 @@
...
+        rdfs:label     "start" ;
+        rdfs:domain     vf:EconomicEvent ;
+        rdfs:comment   "The starting date and optionally time of the economic event." .
+
+vf:duration a owl:ObjectProperty ;
+        rdfs:label     "duration" ;
+        rdfs:domain     vf:EconomicEvent ;
+        rdfs:comment   "The observed duration of the economic event." .
+
+vf:image a owl:DatatypeProperty ;
```
Comment (djodjoni) [inline release-doc-in-process/all_vf.TTL]: use foaf?

File: release-doc-in-process/all_vf.TTL [hunk 10]
```diff
@@ -6,255 +6,854 @@
...
+        rdfs:range      qudt:QuantityValue ;
+        rdfs:comment    "The quantity and unit of the event. This is the quantity that could be used to increment or decrement a resource, depending on the type of resource and type of event." .
+
+vf:currentLocation a owl:ObjectProperty ;
+        rdfs:label   "current location" ;
+        rdfs:domain  vf:EconomicResource ;
+        rdfs:range   vf:Location ;
+        rdfs:comment        "The current place an economic resource is located.  Could be at any level of granularity, from a town to an address to a warehouse location.  Should be mappable." .
+
+vf:primaryLocation a owl:ObjectProperty ;
```
Comment (djodjoni) [inline release-doc-in-process/all_vf.TTL]: foaf or shema org?

**Comment by djodjoni** [comment]
@fosterlynn  I can add a travis/codeship script and a jar to automate the creation of html, rdf formats and vowl json, where failure to generate those will be noted when a PR is made so we can make sure there is clean convertible ontology. Also some MDs/gitbook pages might be generated depending on the comment/description in the ttl. 
I can make a separate PR for that actually

File: release-doc-in-process/all_vf.TTL (hunk 4 above)
Comment (fosterlynn) [inline release-doc-in-process/all_vf.TTL]: seems to be ok now; must have fixed earlier, possibly by accident :)

**Comment by fosterlynn** [comment]
>+vf:trackingIdentifier a owl:DatatypeProperty ;

>DatatypeProperty could limit future uses, as ID could be represented by an individual resource. something like skos labels had suffered from and skos-xl emerged to solve this.

This is interesting, I'm not sure how to think about it.  In that case should we use owl:Thing?

**Comment by fosterlynn** [comment]
>@fosterlynn I can add a travis/codeship script and a jar to automate the creation of html, rdf formats and vowl json, where failure to generate those will be noted when a PR is made so we can make sure there is clean convertible ontology.

Not a bad idea.  Although right now webvowl generator has been down all morning, as far as I can tell.  Do you think it would be too limiting?  Especially if people are working in branches?  I don't feel qualified to judge this.

**Comment by djodjoni** [comment]
> This is interesting, I'm not sure how to think about it. In that case should we use owl:Thing?

Rather ObjectProperty

> Although right now webvowl generator has been down all morning, as far as I can tell. Do you think it would be too limiting?

I use the java library the web tool is using so i test offline and it also spits out nice readable errors :)
This in the java ugly thing i mentioned before :)

About branches that doesn't matter. The generated files might be pushed/uploaded after merging

**Comment by fosterlynn** [comment]
>>This is interesting, I'm not sure how to think about it. In that case should we use owl:Thing?

>Rather ObjectProperty

But for our known use cases, it is a string - a lot number or serial number....

>The generated files might be pushed/uploaded after merging

Maybe that is better.  I set it up to generate each time.  But I don't know how that will behave inside of gitbook.... haven't tested that way.

**Comment by elf-pavlik** [comment]
I think we can merge this one and continue in more atomic PRs that will follow up

**Comment by fosterlynn** [comment]
>I think we can merge this one and continue in more atomic PRs that will follow up

Yeah, I think that will be easier... I'll pull the specific review comments from @djodjoni for the foaf issue (those are the ones not dealt with yet).

**Comment by djodjoni** [comment]
:+1:  del the branch ? :)

**Pull request merged**